DigitalExcellence/dex-frontend
# Implemented alert service & component

## Description

Implemented alert service & displaying of alerts

## Type of change


## Checklist


## Steps to Test or Reproduce
1. Open home.component.ts
2. Uncomment lines 38 - 44 & save
3. Alerts should no be displayed

## Link to issue

Closes: #76
